### ISSUE TITLE
Ensure subscribe happens after node has started and settled down

### ISSUE THREAD

**Issue by oskarth** [status-im/nim-waku#227]
# Problem

See https://github.com/status-im/nim-waku/pull/226 for context.

Subscribe to default topics should happen after node has started and settled down. https://github.com/status-im/nim-waku/commit/03895ffaf559c9c3f8f3cc360dd9af746a3d9ed8

For now, doing this with a sleep 2-5s seems OK.

## Acceptance criteria

Scenario described in https://github.com/status-im/nim-waku/pull/226 and https://github.com/status-im/nim-waku/issues/221 works for wakunode2 _without_ calling rpc_subscribe.

Basically what https://github.com/status-im/nim-waku/pull/152 should've fixed (unclear if regression or untested, perhaps due to use of floodsub by default back then).

## Future enhancements

Instead of sleeping either:

- Make it wait for the right event (meshIsReady or whatever), or
- Ensure it can be done in any order (what vs how)

Neither is needed for now, but would be a more correct way of doing it.

**Comment by oskarth** [comment]
As part of this it'd be good to have a testing scenario issue cut out as described here https://github.com/status-im/nim-waku/issues/221 that would've caught this issue.
